.buildkite/others.rayci.yml

```diff
@@ -10,7 +10,6 @@ steps:
   - label: ":tapioca: build: pip-compile dependencies"
     instance_type: small
     commands:
-      - rm ./python/requirements_compiled.txt
```

Comment:
could you leave a comment here? I think the intention for removing `python/requirements_compiled.txt` is that it will capture upgrades.